packages/sui-pde/src/hooks/platformStrategies.js

```diff
@@ -7,7 +7,14 @@ const getServerStrategy = () => ({
     return pde.getVariation({pde, name: experimentName, attributes})
   },
   trackExperiment: () => {},
-  getForcedValue: () => {}
+  getForcedValue: ({key, queryString}) => {
+    if (!queryString) {
```

Comment:
If there is no `queryString` we will do nothing just like before. No breaking change here!